stable/enterprise/tests/__snapshot__/configmap_test.yaml.snap

```diff
@@ -8,6 +8,7 @@ should render the configmaps:
           clamav:
             db_update_enabled: true
             enabled: <ALLOW_API_CONFIGURATION>
+            max_scan_time: 600000
```

Comment:
With the latest code change I'm making the new default will be 180000 
 (= 30 minutes) rather than 600000 (= 10 minutes).
 
 Probably the helm chart should have that as default too?